projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -16,6 +16,12 @@
 </ng-container>
 
 <ng-template #carouselItem let-item="item">
+  <div
+    id="data-cx-merchandising-product-{{ item.code }}"
```

Comment:
is this used for something?

Reply:
Yea, this is part of the metadata for profile-tag to scrape. Do you prefer that ids are not used?

Reply:
it is fine :)